From 3267ab4d61dc71068f18d5f7c29bf3b67690cc77 Mon Sep 17 00:00:00 2001
From: smplayer-dev <10128951+smplayer-dev@users.noreply.github.com>
Date: Mon, 1 Sep 2025 13:15:45 +0200
Subject: [PATCH] Enable HDPI_SUPPORT

---
 src/smplayer.pro | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/smplayer.pro b/src/smplayer.pro
index e283499ef..29c481063 100644
--- a/src/smplayer.pro
+++ b/src/smplayer.pro
@@ -51,8 +51,7 @@ DEFINES += MPLAYER_SUPPORT
 #DEFINES += SHARE_ACTIONS
 #DEFINES += SHARE_WIDGET
 
-# If Qt >= 5.4
-greaterThan(QT_MAJOR_VERSION, 4):greaterThan(QT_MINOR_VERSION, 3) {
+greaterThan(QT_VERSION, 0x050400) {
 	DEFINES += HDPI_SUPPORT
 }
 
