From 90778a98be1805369fd637cdd0afa0683d851e56 Mon Sep 17 00:00:00 2001
From: smplayer-dev <10128951+smplayer-dev@users.noreply.github.com>
Date: Mon, 1 Sep 2025 13:08:39 +0200
Subject: [PATCH] Fix installation of the qm files

---
 src/smplayer.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/smplayer.pro b/src/smplayer.pro
index b25a73595..e283499ef 100644
--- a/src/smplayer.pro
+++ b/src/smplayer.pro
@@ -73,7 +73,7 @@ win32 {
     QMAKE_POST_LINK += $$quote(cmd /c copy /y $$shell_quote($$replace(QM_SRC_DIR,"/","\\")\\)*.qm $$shell_quote($$replace(QM_DST_DIR,"/","\\")))$$escape_expand(\n\t)
     QMAKE_CLEAN += $${QM_DST_DIR}/*.qm
 } else {
-    QM_SRC_DIR = $${OUT_PWD}
+    QM_SRC_DIR = $${OUT_PWD}/.qm
     QM_DST_DIR = $${OUT_PWD}/translations
     QMAKE_POST_LINK += mkdir -p $${QM_DST_DIR}$$escape_expand(\n\t)
     QMAKE_POST_LINK += cp $${QM_SRC_DIR}/*.qm $${QM_DST_DIR}$$escape_expand(\n\t)
