diff -uraN smplayer-23.6.0/src/main.cpp omv-smplayer-23.6.0/src/main.cpp
--- smplayer-23.6.0/src/main.cpp	2023-06-11 16:25:08.000000000 +0200
+++ omv-smplayer-23.6.0/src/main.cpp	2023-07-24 19:12:57.154574634 +0200
@@ -66,6 +66,7 @@
 	#endif
 #endif
 
+	qputenv("QT_XCB_GL_INTEGRATION", "xcb_egl"); // Needs to be done before the QApplication constructor loads the plugins, needed for VA-API to work
 #ifdef HDPI_SUPPORT
 	QString hdpi_config_path = hdpiConfig();
 	HDPISupport * hdpi = 0;
