--- smplayer-0.6.8/Makefile.orig	2009-03-11 03:21:41.000000000 +0100
+++ smplayer-0.6.8/Makefile	2009-07-26 09:00:01.000000000 +0200
@@ -41,7 +41,7 @@
 
 src/smplayer:
 	./get_svn_revision.sh
-	+cd src && $(QMAKE) $(QMAKE_OPTS) && $(DEFS) make
+	+cd src && $(QMAKE) "QMAKE_CFLAGS+=$(CFLAGS)" "QMAKE_CXXFLAGS+=$(CXXFLAGS)" "QMAKE_LFLAGS+=$(LDFLAGS)" $(QMAKE_OPTS) && $(DEFS) make
 	cd src && $(LRELEASE) smplayer.pro
 
 clean:
