--- smplayer-17.3.0/Makefile.0000~	2017-03-12 12:25:29.884594560 +0100
+++ smplayer-17.3.0/Makefile	2017-03-12 12:27:09.276435243 +0100
@@ -25,7 +25,7 @@ all: src/smplayer webserver/simple_web_s
 
 src/smplayer:
 	./get_svn_revision.sh
-	+cd src && $(QMAKE) $(QMAKE_OPTS) && $(DEFS) make
+	+cd src && $(QMAKE) "QMAKE_CFLAGS+=$(CFLAGS)" "QMAKE_CXXFLAGS+=$(CXXFLAGS)" "QMAKE_CXXFLAGS_RELEASE=" "QMAKE_LFLAGS+=$(LDFLAGS)" $(QMAKE_OPTS) && $(DEFS) make
 	cd src && $(LRELEASE) smplayer.pro
 
 webserver/simple_web_server:
